### ISSUE TITLE
No Longer Works Correctly

### ISSUE THREAD

**Issue by Plurmo** [daijro/SearchifyX#2]
I opened the program a few days ago, but it suddenly didn't work. I reopened it a few days later (today) and the issue still persists.

https://user-images.githubusercontent.com/51093971/162644305-3d7c1229-f7db-43b9-bc84-03e1bfe509c2.mp4

**Problems:**
1. The OCR is completely messed up; if I scan text, it's gibberish, if I scan nothing, it's gibberish. I use Tesseract OCR in another program and it works fine.

2. The search is completely messed up. All results appear usually in under one second and are wrong. Sometimes it shows answers with really low confidence, other times it shows nothing. I have all the search engines (quizlet, quizizz, brainly) selected and even tried using different combinations just in case it worked.

**Testing on another computer:**
I had a friend test out the program on their PC and the OCR worked correctly. But, their answer results displayed either low confidence results or nothing, just like in my case.

**Testing on virtual machine:**
It is a fresh windows 10 virtual machine. I ran the program and the OCR worked correctly initially, but after a few scans, it stopped working correctly. It is unclear what caused this. The results also showed low confidence answers or nothing.

I tried restarting my PC. I tried fresh copies of both portable and installer versions.

**BASICALLY:**
OCR might have some issues
Answers no longer work

**Comment by daijro** [comment]
Hello, here are a few things I've noticed in my testing:
1. The searcher broke around a week ago when Bing updated their website. I had a fix already planned for this issue, but Bing seemed to have reverted to its functional state, so I scraped the patch I was working on. I was initially reluctant to switch from Bing due to its low latency time, but it seems to be very unstable lately. I will be sure to add multiple search engines as a fallback!
2. The OCR has always had a high degree of instability across different devices- not sure why since I was never able to reproduce the error myself. I believe there's an issue with the way images are cropped and fed into pytesseract. I will look into it.

Thanks!

Edit: Everything has been fixed in the latest release
